spec/services/session_encryptor_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+describe SessionEncryptor do
+  describe '#load' do
```

Comment:
this is probably a dumb question, but why not name this `decrypt` and the other `encrypt`? As someone who doesn't have a ton of experience with encryption, these method names (`load` and `dump`) are not obvious to me, but `encrypt` and `decrypt` would be